layouts/partials/article/editorial-tools.html

```
@@ -0,0 +1,148 @@
+{{/* Right Editorial Tools sidebar (US-030)
+     Groups: Table of Contents / AI Companion / Annotations / Share Scroll
+     Rendered inside aside.article-tools (≥1280px sticky right column)
+     Uses .reading-companion DOM structure so reading-companion.js auto-initialises.
+*/}}
+
+{{- $page := . }}
+{{- $headers := findRE "<h[1-6].*?>(.|\n])+?</h[1-6]>" .Content }}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)